PowerPointLabs/Test/UnitTest/HighlightLab/BaseHighlightLabTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using Test.Util;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace Test.UnitTest.HighlightLab
+{
+    [TestClass]
+    public class BaseHighlightLabTest : BaseUnitTest
```

Comment:
Okay, I'll remove it for now. Can always add it back later, not too much trouble